apps/rewards/app/components/App/App.js

```diff
@@ -32,16 +33,19 @@ class App extends React.Component {
     refTokens: PropTypes.array.isRequired,
     amountTokens: PropTypes.array.isRequired,
     claims: PropTypes.array.isRequired,
+    tabs: PropTypes.array.isRequired,
```

Comment:
We need to take type checking seriously, From now on, I personally will not consider any PR completed unless their PropTypes are correctly shaped.